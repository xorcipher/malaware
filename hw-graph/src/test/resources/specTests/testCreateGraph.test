# Tests that a graph can be created.

# Creates a new, empty graph.

CreateGraph graph