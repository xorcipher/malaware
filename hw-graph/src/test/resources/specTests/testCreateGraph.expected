# Tests that a graph can be created.

# Creates a new, empty graph.

created graph graph